tests/snappi_tests/multidut/ecn/test_multidut_ecn_marking_with_snappi.py

```diff
@@ -0,0 +1,107 @@
+from tests.common.fixtures.conn_graph_facts import conn_graph_facts, fanout_graph_facts_multidut         # noqa: F401
+from tests.common.snappi_tests.snappi_fixtures import snappi_api_serv_ip, snappi_api_serv_port, \
+    snappi_api, snappi_dut_base_config, get_snappi_ports, get_snappi_ports_for_rdma, cleanup_config, \
+    is_snappi_multidut, get_snappi_ports_multi_dut   # noqa: F401
+from tests.common.snappi_tests.qos_fixtures import prio_dscp_map, \
+    lossless_prio_list   # noqa F401
+from tests.snappi_tests.variables import MULTIDUT_PORT_INFO, MULTIDUT_TESTBED
+from tests.snappi_tests.multidut.ecn.files.multidut_helper import run_ecn_marking_port_toggle_test
+from tests.common.snappi_tests.common_helpers import packet_capture # noqa F401
+from tests.common.snappi_tests.snappi_test_params import SnappiTestParams
+logger = logging.getLogger(__name__)
+pytestmark = [pytest.mark.topology('multidut-tgen')]
+
+
+@pytest.mark.parametrize("multidut_port_info", [
+    pytest.param(
+        multidut_port_info,
+        id=f"multidut_port_keys={'_'.join(multidut_port_info.keys())}"
+    )
+    for multidut_port_info in MULTIDUT_PORT_INFO[MULTIDUT_TESTBED]
+])
+def test_ecn_marking_port_toggle(
+                                snappi_api,                       # noqa: F811
+                                conn_graph_facts,                 # noqa: F811
+                                fanout_graph_facts_multidut,               # noqa: F811
+                                duthosts,
+                                lossless_prio_list,     # noqa: F811
+                                get_snappi_ports,     # noqa: F811
+                                tbinfo,      # noqa: F811
+                                multidut_port_info,     # noqa: F811
+                                prio_dscp_map):                    # noqa: F811
+    """
+    Verify ECN marking both pre and post port shut/no shut toggle
+
+    Args:
+        request (pytest fixture): pytest request object
+        snappi_api (pytest fixture): SNAPPI session
+        conn_graph_facts (pytest fixture): connection graph
+        fanout_graph_facts (pytest fixture): fanout graph
+        duthosts (pytest fixture): list of DUTs
+        lossless_prio_list (pytest fixture): list of all the lossless priorities
+        prio_dscp_map (pytest fixture): priority vs. DSCP map (key = priority).
+        prio_dscp_map (pytest fixture): priority vs. DSCP map (key = priority).
+        tbinfo (pytest fixture): fixture provides information about testbed
+        get_snappi_ports (pytest fixture): gets snappi ports and connected DUT port info and returns as a list
+    Returns:
+        N/A
+    """
+
+    for testbed_subtype, rdma_ports in multidut_port_info.items():
+        tx_port_count = 2
+        rx_port_count = 1
+        snappi_port_list = get_snappi_ports
+        pytest_assert(MULTIDUT_TESTBED == tbinfo['conf-name'],
```

Comment:
This line needs to be removed.  as T0/T1 doesn't need to check this.

Reply:
This is gone due to PR 15099 changes